pkg/adapter/adapter_test.go

```diff
@@ -0,0 +1,688 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package gnmi
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"reflect"
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+
+	"github.com/damianoneill/net/v2/netconf/ops/mocks"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/damianoneill/net/v2/netconf/ops"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/openconfig/gnmi/value"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	pb "github.com/openconfig/gnmi/proto/gnmi"
```

Comment:
i find `pb` a particularly meaningless and unintuitive alias for `gnmi` - I'd rather just see 'gnmi` used.

Reply:
Done, after renaming adapter package from gnmi to adapter.